content/sensu-go/6.2/observability-pipeline/observe-entities/monitor-external-resources.md

```diff
@@ -449,6 +475,11 @@ Use sensuctl to confirm that Sensu is monitoring docs.sensu.io, packagecloud.io,
 
 {{< code shell >}}
 sensuctl event list
+{{< /code >}}
+
+The sensuctl response should be similar to:
+
```

Comment:
I can't seem to get the check to work on the three proxy entities. I see the entities and the check, and I've started the agent with the `--subscriptions proxy` flag, but I can't get the check to produce any events.